setup.cfg

```diff
@@ -8,6 +8,6 @@ exclude =
 
 [tool:pytest]
 addopts = --flake8
-flake8-ignore = E133 E226 E228 N802 N803 N806
+flake8-ignore = E133 E226 E228 N802 N803 N806 E266
```

Comment:
[tool:pytest] allows us to exclude particular error codes for particular files; use use this, as per https://community.lsst.org/t/task-documentation-and-the-numpydoc-flake8-transitions e.g.

I believe the correct thing to add, instead of the E266 above, is the following after `tests/data/*`:

    astrometry.py E266
    fitTanSipWcs.py E266
    ref_match.py E266

which I figured out by removing the E266 exclusion from flake8 (earlier in setup.cfg) and then running flake8.

I'm glad you are fixing this -- I don't know how I missed it; for some reason scons tests works on my Mac and Jenkins without the explicit excludes, but I think they ought to be necessary.